### PR TITLE
Fix broken configuration documentation page

### PR DESCRIPTION
**Pull request by arthurhammer**

### CODE REVIEW

File: site/_docs/configuration.md [hunk 1]
```diff
@@ -351,27 +351,27 @@ before your site is served.
...
 For example, suppose you set this conditional statement in your code:
 
-```liquid
- {% raw %}
- {% if jekyll.environment == "production" %}
-   {% include disqus.html %} 
- {% endif %}
- {% endraw %}
-```
+{% highlight ruby %}
```
Comment (parkr) [inline site/_docs/configuration.md]: Why did you choose ruby here? I believe this should still be Liquid.

**Comment by parkr** [comment]
Thanks so much, @arthurhammer!

File: site/_docs/configuration.md (hunk 1 above)
Comment (arthurhammer) [inline site/_docs/configuration.md]: @parkr I wasn't sure. I looked at the templates page and it [used `ruby` and `text`](https://github.com/jekyll/jekyll/blame/master/site/_docs/templates.md#L270) for Liquid highlighting. Should I update?

**Comment by arthurhammer** [comment]
@parkr Changed it to use Liquid highlighting now.

**Comment by envygeeks** [comment]
@arthurhammer can you please also change "bash" to "shell", while I love bash with all my heart shell is the preferred shell for simple scripts just to be sure.

**Comment by arthurhammer** [comment]
@envygeeks Will do. Should I squash these three into one commit?

**Comment by envygeeks** [comment]
@arthurhammer yes, please that would be lovely.

**Comment by parkr** [comment]
W squash, :shipit:

**Comment by arthurhammer** [comment]
@envygeeks Change to "shell" as in `{% highlight shell %}`? There doesn't appear to be a shell highlighter: `ClassNotFound: no lexer for alias 'shell' found`.

**Comment by envygeeks** [comment]
I think it's just `sh` sorry.

**Comment by arthurhammer** [comment]
All should be well now, thanks!

**Comment by envygeeks** [comment]
:heart:

**Pull request merged**